processor/trilep_analysis.py

```diff
@@ -769,7 +802,7 @@ def postprocess(self, accumulator):
                     "dilepton_mass_XG": hist.Hist("Counts", dataset_axis, eft_axis, pred_axis, systematic_axis, mass_axis),
                     "dilepton_mass_ttZ": hist.Hist("Counts", dataset_axis, eft_axis, pred_axis, systematic_axis, mass_axis),
                     "dilepton_mass_topW": hist.Hist("Counts", dataset_axis, eft_axis, pred_axis, systematic_axis, mass_axis),
-                    "signal_region_topW": hist.Hist("Counts", dataset_axis, eft_axis, pred_axis, systematic_axis, sr_axis, charge_axis, nossf_axis),  # NOTE this will also need the EFT axis
+                    "signal_region_topW": hist.Hist("Counts", dataset_axis, eft_axis, pred_axis, systematic_axis, sr_axis, charge_axis, nossf_axis, eft_axis),
```

Comment:
does it not complain about the two EFT axis?

Reply:
It didn't, I just realized there's two... thank you!